### PR TITLE
PS-6783 [Doc] Release Notes

### PR DESCRIPTION
**Pull request by patrickbirch**
Release notes for 8.0.19-10.

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-8.0.19-10.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.. _PS-8.0.19-10:
+
+================================================================================
+*Percona Server for MySQL* 8.0.19-10
+================================================================================
+
+:Date: March 11, 2020
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.19-10.rst]: date changed

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-8.0.19-10.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.19-10.rst]: The date is based on the Release Management Dashboard. When the date is updated, the change is propogated to the release note by the Jira-bot. At this point, I'm manually changing the date.

File: doc/source/release-notes/Percona-Server-8.0.19-10.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* :jirabug:`PS-4464`: Exposed the last global transaction identifier (GTID) executed for a CONSISTENT SNAPSHOT.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-6775`: Removed the KEYRING_ON option from the default_table_encryption.
+* :jirabug:`PS-6733`: Added binary search to the Data masking plugin.
+
```
Comment (robgolebiowski) [inline doc/source/release-notes/Percona-Server-8.0.19-10.rst:30]: Please add here PS-5729 here, I have forgotten to set status to fixed ...

File: doc/source/release-notes/Percona-Server-8.0.19-10.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6811`: Service failed to start asserting ACL_PROXY_USER::check_validity if --skip-name-resolve=1 and there is a Proxy user. (Upstream :mysqlbug:`98908`)
```
Comment (robgolebiowski) [inline doc/source/release-notes/Percona-Server-8.0.19-10.rst:36]: Please add here PS-6106 here, I have forgotten to set status to fixed ...

**Review by robgolebiowski — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-8.0.19-10.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.19-10.rst:30]: Verified status is fixed. Thanks, Robert.

File: doc/source/release-notes/Percona-Server-8.0.19-10.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.19-10.rst:36]: Verified status is fixed.

**Review by robgolebiowski — APPROVED** [review]

**Review by kamil-holubicki — APPROVED** [review]

**Review by inikep — APPROVED** [review]

**Pull request merged**